web/src/components/Nav.jsx

```diff
@@ -34,22 +33,9 @@ export default function PrimarySearchAppBar({ setSearchInput, searchInput }) {
               </Typography>
             </NavLink>
             <img src={ComingSoon} alt="Coming Soon!!" style={{height: 50, marginLeft: 50, marginRight: 20}} />
+            {matchSearchBar.isExact ? <SearchBar setSearchInput={setSearchInput} searchInput={searchInput} /> : null}
+            {!matchGamesPage.isExact ? console.log("found it") : null}
```

Comment:
Is this a debug code that's left here?

Reply:
Thanks for catching that - that's old code. I've pushed a new commit that matches what I showed you on slack